db/fixtures/ae_datastore/ManageIQ/System/Event/EmsEvent/RHEVM.class/user_remove_vm_finished.yaml

```diff
@@ -9,8 +9,8 @@ object:
     description: 
   fields:
   - rel4:
-      value: "/System/event_handlers/src_vm_disconnect_storage"
-  - rel5:
       value: "/System/event_handlers/event_action_policy?target=src_vm&policy_event=vm_unregister&param="
+  - rel5:
+      value: "/System/event_handlers/event_action_refresh_sync?target=src_vm"
```

Comment:
@Ladas it didn't used to take into account the disks though, it would unconditionally disconnect the storage